### ISSUE TITLE
Search in diff view

### ISSUE THREAD

**Issue by jankuca** [git-up/GitUp#190]
It would be quite useful to be able to search the diff of an open commit. Especially when splitting large commits, it is easy to overlook related chunks of code.

**Comment by comatory** [comment]
Yes please this! I don't want to use another tool just for searching commits.
